components/Question/Header/index.tsx

```diff
@@ -1,17 +1,31 @@
+import { useContext } from 'react';
+import ProgressBar from '@ramonak/react-progress-bar';
+
 import Text from '@/components/common/Text';
-import { TEXTS } from './index.constants';
+import { QuestionContext } from '@/contexts/QuestionContext';
+import { Questions } from '../index.constants';
 import * as Styled from './index.styles';
 
 const Header = () => {
+  const { currentQuestion } = useContext(QuestionContext);
+
   return (
     <Styled.Wrapper>
-      {Object.values(TEXTS).map(({ title, description }) => (
-        <Text as="p" variant="heading" key={title}>
-          <span>{title}</span>
-          <br />
-          <span>{description}</span>
+      <Styled.InnerWrapper>
+        <Text variant="caption">&lt; 뒤로</Text>
+        <Text fontWeight={700}>고전겜BTI</Text>
+        <Text variant="caption">
+          {currentQuestion} / {Questions.length}
```

Comment:
대충은 알겠지만, 계산식이 된 변수로 보여주면 더 깔끔할듯
예를 들어,
```suggestion
          {getProgress()}
```